classes/class-wc-connect-payment-methods-store.php

```diff
@@ -47,6 +47,7 @@ public function fetch_payment_methods_from_connect_server() {
 			$this->update_payment_methods( $payment_methods );
 
 			$this->potentially_update_selected_payment_method_from_payment_methods( $payment_methods );
+			return true;
```

Comment:
Let's explicitly return `false` from the two `WP_Error` cases. Also, please add a PHP Docblock for this function. 🙂